### PR TITLE
Paystack Integration

### PR DESCRIPTION
**Pull request by meeklife**

### CODE REVIEW

File: apps/orders/views.py [hunk 1]
```diff
@@ -54,44 +59,118 @@ def checkout(self, request):
...
-        ShoppingCart.objects.filter(user=request.user).delete()
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        serializer = self.get_serializer(order)  # noqa
 
-    @action(detail=True, methods=['post'])
-    @transaction.atomic
-    def process_payment(self, request, pk=None):
-        order = self.get_object()
+        kobo_amount = int(total_cost * 1000)
+        # callback_url = request.build_absolute_uri(reverse("payment_verify"))
```
Comment (kryzbone) [inline apps/orders/views.py:65]: is this suppose to be 100 or 1000

File: apps/orders/views.py [hunk 2]
```diff
@@ -54,44 +59,118 @@ def checkout(self, request):
...
+        callback_url = request.build_absolute_uri(
+            "http://127.0.0.1:8000/api/order/payment/verify/"
+        )
+        # callback_url = reverse("payment_verify")
```
Comment (kryzbone) [inline apps/orders/views.py:70]: why se this instead of `reverse`?

**Review by kryzbone — CHANGES_REQUESTED** [review]

File: apps/orders/views.py (hunk 1 above)
Comment (meeklife) [reply apps/orders/views.py:65]: fixed

File: apps/orders/views.py (hunk 2 above)
Comment (meeklife) [reply apps/orders/views.py:70]: I was getting some errors with reverse

Comment (meeklife) [reply apps/orders/views.py:70]: I'm still getting the "NoreverseMatch" error

**Review by kryzbone — APPROVED** [review]

**Pull request merged**